src/test/java/com/connexta/store/controllers/StoreControllerTest.java

```diff
@@ -32,11 +33,13 @@
 
   @Mock private StoreService mockStoreService;
 
+  @Mock private IngestService mockIngestService;
+
   private StoreApi storeApi;
 
   @BeforeEach
   public void beforeEach() {
-    storeApi = new StoreController(mockStoreService, STORE_API_VERSION);
+    storeApi = new StoreController(mockIngestService, mockStoreService, STORE_API_VERSION);
```

Comment:
It might be possible to inject the ingest and store services as `@MockBean`, and then `@Inject` to inject the store controller instead of calling its constructor.